### PR TITLE
[AC-4071] feat: update bot profile style for favicon case

### PR DESCRIPTION
**Pull request by bang9**
## Changes
- update bot profile style for favicon case

ticket: [AC-4071]

## Additional Notes
- figma: https://www.figma.com/design/KgpsIWkbP3qHdPgRgJaRPW/2409_Product-bumper(Onboarding%2FDashboard)_WIP?node-id=905-24921&node-type=frame&m=dev
- image

## Checklist
Before requesting a code review, please check the following:
- [x] **[Required]** CI has passed all checks.
- [x] **[Required]** A self-review has been conducted to ensure there are no minor mistakes.
- [x] **[Required]** Unnecessary comments/debugging code have been removed.
- [x] **[Required]** All requirements specified in the ticket have been accurately implemented.
- [x] Ensure the ticket has been updated with the sprint, status, and story points.

[AC-4071]: https://sendbird.atlassian.net/browse/AC-4071?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/BotProfileImage.tsx [hunk 1]
```diff
@@ -4,9 +4,35 @@ import { useTheme } from 'styled-components';
...
 import { getColorBasedOnSaturation } from '../colors';
 import { useConstantState } from '../context/ConstantContext';
+import { themedColors } from '../foundation/colors/css';
 import BotProfileIcon from '../icons/bot-profile-image-small.svg';
 
-const Container = styled.span<{ backgroundColor: string; size: number }>`
+function isMaybeFavicon(url: string) {
+  if (url.length < 4) return false;
+  const fileName = url.substring(url.lastIndexOf('/') + 1);
+  return /fav_|favicon|\.ico/.test(fileName);
```
Comment (chohongm) [inline src/components/BotProfileImage.tsx]: fav_|favicon these two are ok? file name could be anything

File: src/main.tsx [hunk 2]
```diff
@@ -8,12 +8,24 @@ const WidgetApp = () => {
   const appId = urlParams.get('app_id') ?? import.meta.env.VITE_CHAT_WIDGET_APP_ID;
   const botId = urlParams.get('bot_id') ?? import.meta.env.VITE_CHAT_WIDGET_BOT_ID;
   const locale = urlParams.get('locale') ?? undefined;
+  const region = urlParams.get('region') ?? undefined;
+
+  function getHost(region?: string) {
+    if (region && region.startsWith('no')) {
```
Comment (chohongm) [inline src/main.tsx:14]: maybe no need to add 
region.startsWith('no')?

**Review by chohongm — APPROVED** [review]
LGTM

File: src/components/BotProfileImage.tsx (hunk 1 above)
Comment (bang9) [reply src/components/BotProfileImage.tsx]: We retrieve information from the customer's site on the Sendbird server.
Therefore, we agreed with the server to add the `fav_` prefix when saving the favicon.

File: src/main.tsx (hunk 2 above)
Comment (bang9) [reply src/main.tsx:14]: I added it because I thought it would be needed to run tests with test region on the latest/PR branch.

**Pull request merged**